.github/dependabot.yml

```diff
@@ -13,3 +13,7 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "yarn"
```

Comment:
FYI not added in #382 https://github.com/jenkinsci/workflow-cps-plugin/blob/cee026aaaab559d970d2b5d444a6a69775196185/.github/dependabot.yml#L5-L7